gef.py

```diff
@@ -9963,12 +9964,15 @@ def tmux_setup(self) -> None:
         forcing the context to be redirected there."""
         tmux = which("tmux")
         ok("tmux session found, splitting window...")
-        old_ptses = set(os.listdir("/dev/pts"))
-        gdb.execute(f"! {tmux} split-window -h 'clear ; cat'")
+        
+        pane_info = subprocess.check_output(["tmux", "splitw", "-h",'-F#{session_name}:#{window_index}.#{pane_index}-#{pane_tty}', "-P"]).decode().strip().split("-")
```

Comment:
this should be `tmux` not `"tmux"`, and double quotes around the long arg.

Reply:
double quotes will not work here